src/bosh_azure_cpi/spec/manual_tests/performance/run_performance.sh

```diff
@@ -0,0 +1,116 @@
+chruby 2.4.4
+source ./run_performance.pri.sh
+
+export SERVICE_HOST_BASE="core.windows.net"
+
+pushd tf
+  terraform init
+  PREFIX=$(echo "$PREFIX" | tr '[:upper:]' '[:lower:]')
+  terraform apply -var prefix="${PREFIX}" \
+  -var subscription_id="${SUBSCRIPTION_ID}" \
+  -var client_id="${CLIENT_ID}" \
+  -var client_secret="${CLIENT_SECRET}"  \
+  -var tenant_id="${TENANT_ID}"  \
+  -var location="westus"
+
+  export RESOURCE_GROUP=${PREFIX}-rg
+  export VNET_NAME=${PREFIX}-vnet
+  export SUBNET_NAME=${PREFIX}-bosh-subnet
+  default_storage_account=$(echo $PREFIX | sed -e 's/[^0-9a-z]//g')
+  default_storage_access_key=$(az storage account keys list -g ${PREFIX}-rg -n ${default_storage_account} | jq .[0].value)
+
+  connection_string="DefaultEndpointsProtocol=https;AccountName=${default_storage_account};AccountKey=${default_storage_access_key};EndpointSuffix=${SERVICE_HOST_BASE}"
+
+  az storage container create --name bosh --connection-string ${connection_string}
+  az storage container create --name stemcell --public-access blob --connection-string ${connection_string}
+  az storage table create --name stemcells --connection-string ${connection_string}
+popd
+
+cat > ./create_db.sql <<EOS
+CREATE USER test WITH PASSWORD 'test';
+CREATE DATABASE testdb;
+GRANT ALL PRIVILEGES ON DATABASE testdb to test;
+EOS
+psql -f ./create_db.sql
+
+cat > ./cpi.cfg <<EOF
+---
+azure:
+  environment: AzureCloud
+  subscription_id: ${SUBSCRIPTION_ID}
+  storage_account_name: $default_storage_account
+  storage_account_type: Standard_LRS
+  resource_group_name: ${RESOURCE_GROUP}
+  tenant_id: ${TENANT_ID}
+  client_id: ${CLIENT_ID}
+  client_secret: ${CLIENT_SECRET}
+  ssh_user: vcap
+  ssh_public_key: $(cat ~/.ssh/id_rsa.pub)
+  default_security_group: ${PREFIX}-bosh-sg
+  debug_mode: false
+  use_managed_disks: true
+  vnet_name: ${VNET_NAME}
+  subnet_name: ${SUBNET_NAME}
+  perform_times: 1
```

Comment:
`vnet_name`, `subnet_name` and `perform_times` should not be part of the CPI global configurations (the properties under `azure`).
I'm OK to put them here just for testing. I also suggest adding a comment here.